index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Mobile Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <header class="first">
+        <img src="/images/Welcome.png" alt="welcome button at the top left">
+        <img src="/images/Union.png" alt="a union of three vectors">
+    </header>
+    <br>
+    <div class="second">
+      <h1>  Hey there. I’m Badru
+I’m a software developer</h1>
+    <p>
+    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+        <img src="/images/social media icons (1).png" alt="social media icons">
+    </div>
```

Comment:
- [ ] please try to change this `<div>` to `<section>`cause it will be more descriptive and also it will improve the SEO benchmarks